app/src/app/admin/products/_components/input-uploader-modal.tsx

```
@@ -0,0 +1,124 @@
+            id='picture'
+            type='file'
+            accept='image/jpeg,image/png,image/webp'
+            multiple
+            max={2}
+            onChange={handleImageSelected}
+          />
+          <p>Total files size: {`${totalSize.value}${totalSize.suffix}`}</p>
+          <i>Max Total file size: 5MB</i>
+          <div className='grid grid-cols-1 gap-1.5 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4'>
+            {previews.map((preview, index) => (
+              <div className='grid grid-cols-3 items-center justify-center gap-2'>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)